algebird-core/src/main/scala/com/twitter/algebird/Ring.scala

```diff
@@ -143,19 +143,19 @@ object FloatRing extends Ring[Float] {
   override def plus(l: Float, r: Float): Float = l + r
   override def minus(l: Float, r: Float): Float = l - r
   override def times(l: Float, r: Float): Float = l * r
-  override def sum(t: TraversableOnce[Float]): Float = {
```

Comment:
Type being erasured is a bummer and this is probably for the best ... however i reckon that at some we should nail these and release a major version. It feels that the API is becoming a bit incoherent in come cases.

Reply:
yeah, maybe we should have an issue to track the wins... that said, I still think minimizing source breaks to removing real bugs is a good goal.